### PR TITLE
Removed run-as-non-root

### PR DESCRIPTION
**Pull request by anneliawa**

### CODE REVIEW

File: pkg/apis/deployment/securitycontext.go [hunk 1]
```diff
@@ -2,40 +2,33 @@ package deployment
...
 }
 
 type securityContextBuilder struct {
 	forceRunAsNonRoot bool
 }
 
-func NewSecurityContextBuilder(forceRunAsNonRoot bool) *securityContextBuilder {
-	return &securityContextBuilder{forceRunAsNonRoot: forceRunAsNonRoot}
+func NewSecurityContextBuilder() *securityContextBuilder {
+	return &securityContextBuilder{forceRunAsNonRoot: true}
```
Comment (satr) [inline pkg/apis/deployment/securitycontext.go:18]: Argument should not be removed

**Review by satr — CHANGES_REQUESTED** [review]

File: pkg/apis/deployment/securitycontext.go (hunk 1 above)
Comment (anneliawa) [reply pkg/apis/deployment/securitycontext.go:18]: should this still not be removed?

Comment (satr) [reply pkg/apis/deployment/securitycontext.go:18]: As it is decided to remove functionality `forceRunAsNonRoot` - this need to be removed (you did this allredy correctly), as well as arguments in logic it used (and env-var)

**Review by satr — CHANGES_REQUESTED** [review]
This can be removed https://github.com/equinor/radix-operator/blob/7ae5c7dbb367b98ca404eba21bde1c2069ab4604/pkg/apis/defaults/environment_variables.go#L91.
Also any use of `RADIX_DEPLOYMENTS_FORCE_NON_ROOT_CONTAINER`

**Review by satr — APPROVED** [review]

**Pull request merged**